pkg/linters/images/rules/images.go

```diff
@@ -358,3 +359,91 @@ type werfFile struct {
 	From     string `json:"from" yaml:"from"`
 	Final    *bool  `json:"final" yaml:"final"`
 }
+
+func checkImageNames(moduleName, path string) errors.LintRuleErrorsList {
+	result := errors.LintRuleErrorsList{}
+
+	imagesPath := filepath.Join(path, ImagesDir)
+
+	if !IsExistsOnFilesystem(imagesPath) {
+		return result
+	}
+
+	err := filepath.Walk(imagesPath, func(fullPath string, _ os.FileInfo, err error) error {
+		if err != nil {
+			return err
+		}
+
+		if fsutils.IsDir(fullPath) {
+			return nil
+		}
+
+		if filepath.Base(fullPath) == "werf.inc.yaml" {
+			lines, err := checkImageNamesFromFile(fullPath)
+			path, _ := filepath.Rel(imagesPath, fullPath)
+			if err != nil {
+				result.Add(errors.NewLintRuleError(
+					ID,
+					path,
+					moduleName,
+					nil,
+					"Cannot read file: %s",
+					err.Error(),
+				))
+				return nil
+			}
+			if len(lines) > 0 {
+				result.Add(errors.NewLintRuleError(
+					ID,
+					path,
+					moduleName,
+					nil,
+					"Image name format should be like `image: {{ .ModuleName }}/{{ .ImageName }}[-something]` in file: %s\n\t\t  broken lines:\n\t\t  %s",
```

Comment:
We print `in file` here. But filename is also duplication in the Object field.
```
Message - Image name format should be like `image: {{ .ModuleName }}/{{ .ImageName }}[-something]` in file: webui/werf.inc.yaml
...
Object  - webui/werf.inc.yaml
```

do we need this duplication?

Reply:
Deleted the file name from the message